.github/workflows/auto-format.yml

```diff
@@ -0,0 +1,28 @@
+name: Auto-Formatting
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - name: Run Spotless
+      run: ./gradlew spotlessApply
+    - name: Commit changes
+      uses: EndBug/add-and-commit@v2.1.0
+      with:
+        author_name: tripleabuilderbot
+        author_email: tripleabuilderbot@gmail.com
+        message: "Auto-Formatting"
+        path: "."
+        pattern: "*.java"
+        force: false
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # Leave this line unchanged
```

Comment:
Where are the secrets defined?
Perhaps it would make sense to do a wiki write-up to document how the github actions/workflow is configured.

Reply:
In the settings of this repo.
`secrets.GITHUB_TOKEN` however is always present, and is an exception to the other secret tokens: It's present even in PRs

Reply:
The documentation can be found here:
https://help.github.com/en/actions/automating-your-workflow-with-github-actions

I can only tell you what I read there...